packages/react-input/src/components/Input/useInputStyles.ts

```diff
@@ -139,7 +139,6 @@ const useRootStyles = makeStyles({
       ...shorthands.borderColor(tokens.colorNeutralStroke1Hover),
       borderBottomColor: tokens.colorNeutralStrokeAccessibleHover,
     },
-    // DO NOT add a space between the selectors! It changes the behavior of make-styles.
     ':active,:focus-within': {
```

Comment:
Note: there is a bug in the version of Griffel currently used in this package that will apply this `:focus-within` style to buttons used in the `contentBefore` and `contentAfter` slots. [The PR that fixes is already merged](https://github.com/microsoft/griffel/commit/7540e722795eea0b027207343c74b1cf8eaec1d8), just need to have a new version published and the dependency updated here.